### PR TITLE
Added X-Settlers

### PR DESCRIPTION
**Pull request by Mailaender**
a native Settlers III clone.

### CODE REVIEW

File: games/x.yaml [hunk 1]
```diff
@@ -83,6 +83,21 @@
...
   url: http://xmoto.tuxfamily.org/
 
+- name: X-Settlers
+  development: active
+  images:
+  - https://raw.githubusercontent.com/Skarmux/X-Settlers/master/xsettlers/docs/screenshots/xsettlers_textured.png
+  lang:
+  - C++
+  license:
+  - Apache-2.0
```
Comment (caiotava) [inline games/x.yaml:93]: The `Apache-2.0` license doesn't exist in the validation schema: https://github.com/opengaming/osgameclones/blob/master/schema/games.yaml#L110

I think you could just use `Apache` in this case, what do you think?

Comment (Mailaender) [reply games/x.yaml]: Yes, fixed it.

**Review by caiotava — APPROVED** [review]

File: games/x.yaml [hunk 2]
```diff
@@ -83,6 +83,22 @@
   updated: 2020-12-16
   url: http://xmoto.tuxfamily.org/
 
+- name: X-Settlers
+  development: active
+  images:
+  - https://raw.githubusercontent.com/Skarmux/X-Settlers/master/xsettlers/docs/screenshots/xsettlers_textured.png
+  lang:
+  - C++
```
Comment (cxong) [inline games/x.yaml:91]:
```suggestion
  - C++
  framework:
  - SDL2
```

Comment (Mailaender) [reply games/x.yaml:91]: Ah, nice spotting.

**Review by cxong — APPROVED** [review]

**Pull request merged**